runtime/doc/eval.txt

```diff
@@ -2476,6 +2477,28 @@ call({func}, {arglist} [, {dict}])			*call()* *E699*
 		{dict} is for functions with the "dict" attribute.  It will be
 		used to set the local variable "self". |Dictionary-function|
 
+capture({command})					*capture()*
+		Capture output of {command}.
+		If {command} is a |String|, it returns {command} output.
+		If {command} is a |List|, it returns concatenated outputs of
+		each item.
+		Examples: >
+			let a = capture('echon "foo"')
+			echo a
+<			foo >
+			let a = capture(['echon "foo"', 'echon "bar"'])
+			echo a
+<			foobar
+		This function is not available in the |sandbox|.
+		Note: The commands are run as if they were prepended with
+		|:silent| modifier.  |:redir| and |capture()| work in parallel:
+		|capture()| does not cancel |:redir|, |:redir| does not cancel
+		|capture()|.  |capture()| inside |capture()| is allowed
+		unlike nested |:redir|s, outer capture will not catch
+		commands' output of the inner one, but inner will neither
+		cancel outer.
+		Note: The attributes are ignored.
```

Comment:
I am not sure that user will understand WTF are “attributes”. Even `:echohl` documentation call this “highlight group”, so better write “Note: highlighting is ignored.”

I would also say that I think that function name is fine, but it is reasonable to mention it under `:h :redir`. It already lists one alternative, you may add more:

> Alternatives are the 'verbosefile' option and the |capture()| function, both can be used in combination with ":redir".